.goreleaser.yml

```diff
@@ -27,6 +30,10 @@ dockers:
 
   - "{{ if not .Prerelease }}ghcr.io/k8up-io/k8up:latest{{ end }}"
   - "{{ if not .Prerelease }}quay.io/k8up-io/k8up:latest{{ end }}"
+  goarch: amd64
+# - <<: *image_template
+#   id: docker-arm64
+#   goarch: arm64
```

Comment:
I think this file is missing all the config for docker manifests. Also the ARM images need their own tags (e.g. suffixed with `-arm`)

See [docker manifest](https://docs.docker.com/engine/reference/commandline/manifest/) how it generally works.
The Gist is basically:
* Create and push dedicated AMD64 and ARM images with tags (e.g. suffixed)
* Create a manifest that combines those tags into one

Essentially, the registry will deliver the suffixed image tags, e.g. `v2.0.0-arm64`, when a client requests `v2.0.0` with an ARM client. The `v2.0.0` tag is basically "virtual", or a placeholder for the actual, arch-dependent image.

Did you test this out?
Did you also look into https://github.com/ccremer/kubernetes-zfs-provisioner/pull/11/files, where I already engineered everything for multi-arch building? Just copy-paste :smile:

Reply:
> Did you test this out?

Not yet, hence also the _Draft_ status of this PR.

> Did you also look into https://github.com/ccremer/kubernetes-zfs-provisioner/pull/11/files, where I already engineered everything for multi-arch building? Just copy-paste 😄

Nice, I did miss that. Thanks!

> I think this file is missing all the config for docker manifests. Also the ARM images need their own tags (e.g. suffixed with -arm)

I did think this was simpler. Thanks for the quick explainer!